third_party/modules/llvm-raw/20240919.0-94c024a/MODULE.bazel

```diff
@@ -0,0 +1,10 @@
+module(
+    name = "llvm-raw",
+    version = "20240919.0-94c024a",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "platforms", version = "0.0.10")
+bazel_dep(name = "zstd", version = "1.5.6", repo_name = "llvm_zstd")
```

Comment:
same for those versions

Reply:
and llvm seems to be using zstd 1.3.0 https://github.com/llvm/llvm-project/blob/6e6d5eae765939cc4074bdd606658e78c4a2a559/utils/bazel/WORKSPACE#L131